golden_response/staging/get_related_locations/county.json

```diff
@@ -1,41 +1,17 @@
 {
   "Count_Person": {
-    "relatedPlaces": [
```

Comment:
This is because we switch to cohort?

Reply:
The related place is dropped from the flume?

Reply:
I think Flume drops relatedLocations property, but that hasn't rolled out.

Reply:
that's interesting.. but this is okay for staging, as I am going to use cohort for related places: https://github.com/datacommonsorg/mixer/pull/329